openapis/swagger.yaml

```diff
@@ -116,6 +137,27 @@ paths:
         - cdis_auth:
             - 'write:index'
             - 'read:index'
+    post:
+      summary: >-
+        Add a new version of the index record associated with the given id
```

Comment:
should add more description about what 'add a new version' means
explain that the content of the data is immutable, if you want to change the content, which means the hashes are changed, then it's a new index document with new uuid